ur_kinematics/src/ur_moveit_plugin.cpp

```diff
@@ -325,6 +325,11 @@ bool URKinematicsPlugin::initialize(const std::string &robot_description,
 
   private_handle.param<std::string>("arm_prefix", arm_prefix_, "");
 
+  // Load the joint names from the group model
+  ur_joint_names_ = joint_model_group->getJointModelNames();
+  ur_link_names_ = joint_model_group->getLinkModelNames();
+
+  /*
```

Comment:
Can the commented out code be removed?  FYI...I prefer `//` instead of `/*`, since it makes the diff more representative of what was changed/commented out (even though I don't like commented out code ;).